src/main/java/org/gridsuite/voltageinit/server/dto/parameters/FilterEquipments.java

```diff
@@ -34,5 +35,9 @@ public class FilterEquipments {
     @Schema(description = "equipments not found in network")
     private List<String> notFoundEquipments;
 
+    @Schema(description = "flag indicating whether the filter still exists in the filter server")
+    @JsonProperty("isValid")
+    private boolean isValid = false;
```

Comment:
I think it is not necessary to add this extra field ?

You could just set the 'filterName' field to null, when the filter does not exist anymore ?

See what is done for the generation dispatch modification, when the filters do not exist anymore : 

https://github.com/gridsuite/network-modification-server/pull/720